src/interfaces.rs

```diff
@@ -24,6 +24,54 @@ impl WebView {
     }
 }
 
+/// Wrapper for `ICoreWebView2_2`.
+#[derive(Clone)]
+pub struct _2 {
```

Comment:
This is super annoying. I'm not really sure what to do about these. It's not actually clear on how to use the suffixed interfaces.

Reply:
Should be similar to this:

https://github.com/sopium/webview2/commit/ca5dbd7959b030ed574b548457c6c16246284a7d#diff-2dc46f450fd671c92abaecb89b2b6842af16747d8a5de2082ae4663c07697b88R136-R153

Once you have a `WebView`, you should be able to use `as_inner` and `get_interface` to obtain an instance of `_2`(`ICoreWebview2_2`).

Reply:
I shall add some more friendly APIs for these later. (And give them some better names...)